R/permillcalculation.R

```diff
@@ -36,27 +36,18 @@ permillcalc <- function(x = fatalencounters, capita = TRUE){
   kdata <- x %>%
     group_by(year) %>%
     count(state) %>%
-    spread(year, n)
-
-  #change NA values (where no data matched) to 0
-  kdata[is.na(kdata)] <- 0
-
-  #Calculate US total
-  kdata["Total" , ][, -1] <- colSums(kdata[, -1])
-  kdata$state <- as.character(kdata$state)
-  kdata[52,1] <- "US"
-
-
-  #add full state names
-  kdata <- cbind(pop_state_and_us[, 1], kdata)
-  colnames(kdata)[1] <- "state_name"
+    spread(year, n) %>%
```

Comment:
spread/gather being replaced by pivot in dplyr, see https://tidyr.tidyverse.org/